recipe/meta.yaml

```diff
@@ -1,16 +1,17 @@
-{% set version = "0.10.5" %}
+{% set version = "0.11.3" %}
 
 package:
   name: pyhdf
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pyhdf/pyhdf-{{ version }}.tar.gz
-  sha256: 6c539080642366c665f4208aa603e31b517f8f28c08ea8cd2bcc57c97faa953e
+  sha256: c3a3bbfbd22c6c97a62b4149f746461b9816d93dd08ce1c9315bd642625426a3
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps -vv  # [not win]
```

Comment:
@ocefpaf Any memory of why this custom install command is only for not Windows? It looks like that would mean it falls back to `build.bat`, right? That looks like it is calling `setup.py` which is definitely not the modern way of doing this:

https://github.com/conda-forge/pyhdf-feedstock/blob/04343148a8e8cfcd1b008ecd1e27ddc8d03525e7/recipe/bld.bat#L9

Reply:
The package still ships a `setup.py` but if you want to try something new please do. I don't have the bandwidth to maintain this package anymore and there are too few users on Windows to make it worth it. Sorry.

Reply:
Please  don't use the `setup.py`. It is only used to build the C extension. Pyhdf have moved to pyproject.toml + setuptools.

Reply:
I'll see what I can do. There were some environment variables set in this custom build script so let's see what happens without it.